### PR TITLE
WT-9244 Document updated verbose levels

### PR DESCRIPTION
**Pull request by korteland**
* Convert all `DEBUG` references to something appropriate
* A couple of minor typo fixes
* Recommend `DEBUG_1` as a default debug level

I rendered this locally and checked it out in a browser, it all seems to render correctly.

### CODE REVIEW

File: src/docs/verbose-messaging.dox [hunk 1]
```diff
@@ -50,9 +58,9 @@ An example configuring a WiredTiger connection with verbosity enabled:
 @snippet ex_verbose.c Configure verbose_messaging
 
 In the above example:
-- \c api is configured to the \c DEBUG verbosity level.
-- \c version is configured to the \c DEBUG verbosity level. If a category is passed without a verbosity
-level (.e.g \c <code>:0</code>), the category will default to the \c DEBUG level (i.e <code>verbose=[version]</code> is equivalent to <code>verbose=[version:1]</code>).
+- \c api is configured to the \c DEBUG_1 verbosity level.
+- \c version is configured to the \c DEBUG_1 verbosity level. If a category is passed without a verbosity
+level (e.g. \c <code>:0</code>), the category will default to the \c DEBUG_1 level (i.e <code>verbose=[version]</code> is equivalent to <code>verbose=[version:1]</code>).
```
Comment (ruby-oujo) [inline src/docs/verbose-messaging.dox:63]: If you are following the style of using '.' after e.g - I would do the same with 'i.e' to 'i.e.' for consistency.

**Review by ruby-oujo — APPROVED** [review]
Had one comment, otherwise LGTM :)

File: src/docs/verbose-messaging.dox (hunk 1 above)
Comment (korteland) [reply src/docs/verbose-messaging.dox]: Thanks, you're right - added in 412b3b8.

**Comment by korteland** [comment]
evergreen merge

**Pull request merged**